src/components/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from './Book';
+
+function BooksList() {
+  const allBooks = useSelector(state => state.book);
+  return (
+    <div>
+      <h2>Books List</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Book ID</th>
+            <th>Title</th>
+            <th>Category</th>
+            <th>  </th>
+          </tr>
+        </thead>
+        <tbody>
+          {allBooks.map(book => (
+            <Book
+              key={book.id}
+              id={book.id}
+              title={book.title}
+              category={book.category}
+            />
```

Comment:
- [ ] The function `handleBookRemove` should be created in this component and passed down to the `Book` component as props instead of calling it directly in the `Book` component. Then you can call the function in the button in your `Book` component ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png) from Pathwright).